core/modules/wmgr/python/dbMgr.py

```diff
@@ -810,7 +944,7 @@ def getIndex(dbName, tblName, chunkId=None):
             if not rows:
                 break
             for row in rows:
-                result.append(list(row))
+                result.append(tuple(row))
 
```

Comment:
Is this documentation enough http://www.slac.stanford.edu/exp/lsst/qserv/2015_05/admin/data-loader.html or do you want internals documented as well?